test/integration/test_rules.py

```diff
@@ -61,6 +61,44 @@ def test_one_column_rule_sql(rule, expected, conn, ctx):
     assert list(expected) == list(results)
 
 
+@pytest.mark.parametrize(
+    "rule, expected",
+    [
+        (GtRule("gt", "value", 4, condition="conditional is TRUE"), [False, False]),
+        (
+            NotNullRule("not_null", "value", condition="conditional is TRUE"),
+            [True, True],
+        ),
+        (GteRule("gte", "value", 4, condition="conditional is TRUE"), [False, True]),
+        (NotRule("not", "value", 4, condition="conditional is TRUE"), [True, False]),
+        (LtRule("lt", "value", 4, condition="conditional is TRUE"), [True, False]),
+        (LteRule("lte", "value", 4, condition="conditional is TRUE"), [True, True]),
+        (EqRule("eq", "value", 4, condition="conditional is TRUE"), [False, True]),
+    ],
+)
+def test_one_column_rule_sql_condition(rule, expected, conn, ctx):
```

Comment:
I like this test of concrete classes. But I guess that this condition should work for any SQLRule we add here, right ? 
what about adding separate test, where you will create own new class of a rule locally, add some sql property to it and test directly that one ?

Should be more helpful when adding new sql rules / changing sql rule - in letting us know if something is broken.

Reply:
Added test for "count" rule.